PairDefination.py

```diff
@@ -0,0 +1,26 @@
+MAJOR_COLORS = ['White', 'Red', 'Black', 'Yellow', 'Violet']
+MINOR_COLORS = ["Blue", "Orange", "Green", "Brown", "Slate"]
+
+def color_pair_to_string(major_color, minor_color):
```

Comment:
Filename is `PairDefinition`. Intuitively, does the `color_pair_to_string` belong in the definition?